tests/neg/i9295b.scala

```diff
@@ -0,0 +1,12 @@
+import scala.language.dynamics
+
+class Foo extends Dynamic {
+  def applyDynamic(arg: Any): Bar = ???
+}
+class Bar extends Dynamic {
+  def applyDynamic(arg: Any)(x: Int): Int = ???
+}
+object F {
+  val foo = new Foo
+  def baz = foo.blah(43) // error: method applyDynamic in class Foo does not take more parameters
```

Comment:
Can we add a positive test here: `foo.blah.blah(43)`?

Reply:
Indeed

Reply:
Actually that one should also fail as it would require a selectDynamic.

Reply:
I added extra tests